File: system-test/dotnet/SystemTests/Legacy/Mod09/SmokeTests/External/ErpSmokeTest.cs
using SystemTests.Legacy.Mod09.Base;
using Xunit;

namespace SystemTests.Legacy.Mod09.SmokeTests.External;

public class ErpSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToErp()
    {
        await Scenario().Assume().Erp().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/optivem/shop/systemtest/legacy/mod09/smoke/external/ErpSmokeTest.java
package com.optivem.shop.systemtest.legacy.mod09.smoke.external;

import com.optivem.shop.systemtest.legacy.mod09.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class ErpSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToErp() {
        scenario.assume().erp().shouldBeRunning();
    }
}


